### PR TITLE
fix: send Spring CSRF as a login response header

### PR DESCRIPTION
**Pull request by Artur-**

### CODE REVIEW

**Review by haijian-vaadin — CHANGES_REQUESTED** [review]
Could there be a test for this?

File: vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinSavedRequestAwareAuthenticationSuccessHandler.java [hunk 1]
```diff
@@ -78,6 +89,15 @@ public void sendRedirect(HttpServletRequest request, HttpServletResponse respons
                     response.setHeader(VAADIN_CSRF_HEADER, csrfToken);
                 }
             }
+            Object springCsrfTokenObject = request.getAttribute("_csrf");
+            if (springCsrfTokenObject != null
```
Comment (joheriks) [inline vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinSavedRequestAwareAuthenticationSuccessHandler.java:93]: Redundant check as `null instanceof X == false`?

Comment (Artur-) [reply vaadin-spring/src/main/java/com/vaadin/flow/spring/security/VaadinSavedRequestAwareAuthenticationSuccessHandler.java:93]: Fixed

**Comment by Artur-** [comment]
Added test

**Review by haijian-vaadin — APPROVED** [review]

**Pull request merged**